harvard-kings-college-london.csl

```diff
@@ -250,8 +250,8 @@
           </group>
         </else-if>
         <else-if type="chapter paper-conference" match="any">
-          <text macro="title" prefix=" '" suffix="', "/>
           <group prefix=" " delimiter=" ">
+            <text macro="title" prefix="'" suffix="',"/>
```

Comment:
could you change this to quotes="true" (the style is already en-GB, so they'll be single quotes with comma outside)